### PR TITLE
docs: sd-hidden-links docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 1]
```diff
@@ -51,65 +50,10 @@ export const Default = {
...
-          </a>
-        </div>
-      </sd-header>
-    </div>`;
-  }
-};
-
-/**
- * "Stack" multiple `sd-hidden-links` by adding multiple of them to the same parent. (Recommended for 2 links)
+ * "Stack" multiple `sd-hidden-links` by adding multiple of them to the same parent.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:53]:
```suggestion
 * Stack multiple `sd-hidden-links` by adding multiple of them to the same parent.
```

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 2]
```diff
@@ -10,14 +10,16 @@ const { generateTemplate } = storybookTemplate('sd-hidden-links');
 const html = String.raw;
 
 /**
- * Can be used to show links only for keyboard users.
+ * Used to show links only for keyboard users.
  *
- * - Per default the hidden links are optimized for a single <sd-button>.
+ * **Related components**:
+ * - [Navigation Item](?path=/docs/components-navigation-item--docs)
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:16]: In figma we also have a template called "Hidden Links"

Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:16]: In figma we also have a template for Navigation

Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:16]:
```suggestion
 * - [Dropdown with Navigation Items](?path=/docs/components-navigation-item--docs)
```

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 3]
```diff
@@ -121,7 +65,10 @@ export const StackLinks = {
 };
 
 /**
- * Use the `sd-hidden-links--multiple` modifier to show mutiple `sd-navigation-item` elements. (Recommended for >2 links)
+ * Use the `&--multiple` class to show mutiple `sd-navigation-item` elements.
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:68]: For the sample the links should be called:

Hidden Link 1
Hidden Link 2
Hidden Link 3

Comment (MartaPintoTeixeira) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts:68]: Same for: Title For Multiple Links

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts (hunk 2 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts]: In a call the three of us concluded that no need for a template for hidden links

Comment (auroraVasconcelos) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts]: https://github.com/solid-design-system/solid/issues/1377

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts (hunk 3 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts:68]: Updated on german level as well

**Comment by Vahid1919** [comment]
Can we move the initial text so that the button does not cover it?
image

**Comment by auroraVasconcelos** [comment]
> Can we move the initial text so that the button does not cover it? image



> Can we move the initial text so that the button does not cover it? image

I can do it. However, from what I know the normal behaviour of an hidden link is exactly to overlay the page content otherwise all website would need space for this in the top.

**Review by Vahid1919 — APPROVED** [review]

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 4]
```diff
@@ -40,16 +39,87 @@ export const Default = {
...
+    </div>`
+};
+
+/**
+ * Use the `&--multiple` class to show mutiple `sd-navigation-item` elements.
+ *
+ * **Hint:**
+ * Recommended for 2 or more links.
+ */
+export const MultipleLinks = {
```
Comment (MartaPintoTeixeira) [inline packages/components/src/styles/hidden-links/hidden-links.stories.ts:73]: Multiple Links samples should say

Hidden Link 1
Hidden Link 2
Hidden Link 3

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts [hunk 5]
```diff
@@ -10,14 +10,16 @@ const { generateTemplate } = storybookTemplate('sd-hidden-links');
 const html = String.raw;
 
 /**
- * Can be used to show links only for keyboard users.
+ * Used to show links only for keyboard users.
```
[the remark this answers is not in the document]
Comment (auroraVasconcelos) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts:13]: Updated

File: packages/components/src/styles/hidden-links/hidden-links.stories.ts (hunk 4 above)
Comment (auroraVasconcelos) [reply packages/components/src/styles/hidden-links/hidden-links.stories.ts:73]: Updated

**Review by MartaPintoTeixeira — APPROVED** [review]

**Pull request merged**